pkg/config/config.go

```diff
@@ -122,4 +126,23 @@ func Get() *IngressConfig {
 		UseSSL:               options.GetBool("UseSSL"),
 		UseClowder:           os.Getenv("CLOWDER_ENABLED") == "true",
 	}
+
+	if os.Getenv("CLOWDER_ENABLED") == "true" {
```

Comment:
Do we want to update the is to the `isClowderEnabled` method while this is open?

Reply:
I think we can do this in a follow up PR imho